blocks/procedures.js

```diff
@@ -996,6 +991,72 @@ Extensions.register(
 /** @this {Block} */
 const procedureCallerGetDefMixin = function() {
   const mixin = {
+    model_: null,
+
+    prevParams_: [],
+
+    argsMap_: new Map(),
+
+    /**
+     * @return {IProcedureModel} The procedure model associated with this
+     *     block.
+     */
+    getProcedureModel() {
+      return this.model_;
+    },
+
+    /**
+     * @param {string} name The name of the procedure model to find.
+     * @param {string[]} params The param names of the procedure model to find.
```

Comment:
`string[]` is not valid Closure type syntax.  Do you mean `Array<string>`?

This provoked a warning from `closure-make-deps`.